e2e-tests/production-runtime/cypress/integration/static-query.js

```diff
@@ -0,0 +1,49 @@
+beforeEach(() => {
+  cy.visit(`/static-query/`).waitForAPIorTimeout(`onRouteUpdate`)
+})
+
+describe(`StaticQuery behavior`, () => {
+  it(`works with inline query`, () => {
+    cy.getTestElement(`inline`)
+      .invoke(`text`)
+      .should(`not.contain`, `Loading`)
+  })
+
+  it(`works with variable query`, () => {
+    cy.getTestElement(`variable`)
+      .invoke(`text`)
+      .should(`not.contain`, `Loading`)
+  })
+
+  it(`works with exported variable query`, () => {
+    cy.getTestElement(`exported`)
+      .invoke(`text`)
+      .should(`not.contain`, `Loading`)
+  })
+
+  describe(`useStaticQuery`, () => {
+    it(`works with inline query`, () => {
+      cy.getTestElement(`use-static-query-inline`)
+        .invoke(`text`)
+        .should(`not.contain`, `Loading`)
+    })
+
+    it(`works with variable query`, () => {
+      cy.getTestElement(`use-static-query-variable`)
+        .invoke(`text`)
+        .should(`not.contain`, `Loading`)
+    })
+
+    it(`works with exported variable query`, () => {
+      cy.getTestElement(`use-static-query-exported`)
+        .invoke(`text`)
+        .should(`not.contain`, `Loading`)
```

Comment:
Silly assertion, but I don't really want to hard code a bunch of data for these tests either. Thoughts?

Reply:
Yeah, I don't like the idea of hard coding data as well. However, this assertion will pass even if functionality breaks and `null` gets rendered for all data.

Reply:
So a breaking version of this is merely rendering a null?

Reply:
If so—I can just guard against that and output “broken” or something if data comes back null

Reply:
It'll actually throw at the moment in `render` because we don't have a `null` check. But yeah, guarding against and asserting that it is not “broken“ seems like a good idea!